src/main/java/pw/kaboom/extras/commands/CommandPing.java

```diff
@@ -0,0 +1,41 @@
+package pw.kaboom.extras.commands;
+
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.craftbukkit.v1_16_R3.entity.CraftPlayer;
+import org.bukkit.entity.Player;
+
+public final class CommandPing implements CommandExecutor {
+
+    public boolean onCommand(final CommandSender sender, final Command command, final String label, final String[] args) {
+        final Player player = (Player) sender;
+        final int ping = ((CraftPlayer) player).getHandle().playerConnection.player.ping;
```

Comment:
Would https://papermc.io/javadocs/paper/1.16/org/bukkit/entity/Player.Spigot.html#getPing-- work here?

Reply:
> Would https://papermc.io/javadocs/paper/1.16/org/bukkit/entity/Player.Spigot.html#getPing-- work here?

Works fine for me. Should be much better than adding an entirely new library.

Reply:
nice